interfaces/builtin/system_observe_test.go

```diff
@@ -109,7 +109,13 @@ func (s *SystemObserveInterfaceSuite) TestMountPermanentPlug(c *C) {
 	tmpdir := c.MkDir()
 	dirs.SetRootDir(tmpdir)
 
-	c.Assert(os.MkdirAll(filepath.Join(tmpdir, "/boot"), 0777), IsNil)
+	// Create a /boot/config-* file so that the interface will generate a bind
+	// mount for it
+	fakeBootDir := filepath.Join(tmpdir, "/boot")
+	c.Assert(os.MkdirAll(fakeBootDir, 0777), IsNil)
+	file, err := os.OpenFile(filepath.Join(fakeBootDir, "config-5.10"), os.O_CREATE, 0644)
+	c.Assert(file.Close(), IsNil)
+	c.Assert(err, IsNil)
```

Comment:
If the error from `OpenFile` is non-nil, the file will be nil and the `Close` will panic
```suggestion
	c.Assert(err, IsNil)
	c.Assert(file.Close(), IsNil)
```